docbook/src/docbkx/autoscale-devguide.xml

```diff
@@ -70,6 +70,16 @@
         </legalnotice>
         <revhistory>
             <revision xmlns:raxm="http://docs.rackspace.com/api/metadata">
+                <date>2015-03-17</date>
+                <revdescription>
+                    <itemizedlist spacing="compact">
+                        <listitem>
+                            <para>Updated <link linkend="identityEndpoints">Identity endpoints </link>section. Changed launchConfiguration loadBalancerId attribute type to "String."</para>
```

Comment:
The space appears to be within the link.

Are there rules about line length any where in DevDoc's guidelines?